sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -666,3 +666,26 @@ def test_rfe_n_features_to_select_warning(ClsRFE, param):
         # larger than the number of features present in the X variable
         clsrfe = ClsRFE(estimator=LogisticRegression(), **{param: 21})
         clsrfe.fit(X, y)
+
+
+def test_rfe_ties_around_threshold():
```

Comment:
This test passes on my system w/o the fix though.

Reply:
I could try to do something similar to the group k fold and have a couple different length tests and see if at least one should fail on your (most?) machines.